src/Client.cpp

```diff
@@ -150,6 +150,13 @@ err_code_t Client::version(types::broadcast_result_t** results, size_t* nHosts)
 }
 
 
+err_code_t Client::quit() {
+  broadcastCommand(keywords::kQUIT, 4);
```

Comment:
Why we pass the number `4`(since it can be computed by `strlen(keywords::kQUIT)`)  explicitly, is this an optimization for performance ?